src/ucp/tag/tag_match.h

```diff
@@ -44,9 +44,14 @@ typedef struct ucp_tag_match {
     size_t                    thresh;         /* All messages above the threshold are posted
                                                  to the transport. Either UCP user buffer
                                                  itself or preregistered internal UCP buffer
-                                                 is posted. */
+                                                 is posted. Messages below the threshold are
+                                                 not posted and will be matched in SW. */
     size_t                    zcopy_thresh;   /* All UCP user buffers above the threshold
-                                                 are posted to the transport. */
+                                                 are posted to the transport. For buffers
```

Comment:
@shamisp this is not exact, how about the following:
`thresh`
Minimal receive buffer size to be passed to tag-matching offload on the transport. This threshold applies both to user-provided buffers and UCP internal buffers.

`zcopy_thresh`
Minimal size of user-provided receive buffer to be passed directly to for tag-matching offload on the transport. Buffers smaller than this threshold would either bounce to UCP internal buffers, or not be used with tag-matching offload at all, according to 'thresh' configuration.

@alex-mikheev we should also put these fields under "offload" structure" or rename them to "offload_thresh", "offload_zcopy_thresh", etc. (i prefer 1st option)